configure

```diff
@@ -2376,7 +2376,7 @@ if test "$disable_opt" = "yes" ; then
   output_sym "CONFIG_DISABLE_OPTIMIZATIONS"
 fi
 if test "$zlib" = "no" ; then
-  echo "Consider installing zlib-dev (zlib-devel), some fio features depend on it."
+  echo "Consider installing zlib-dev (zlib-devel or zlib-static), some fio features depend on it."
```

Comment:
@brycepg I'm guessing to use zlib-static you would have to have zlib-devel too (-devel still provides the headers) so zlib-static isn't really an alternative but more of a complement...

Reply:
It is a bit unclear and, again, RH centric. I've pushed an improvement.